manifests/init.pp

```diff
@@ -10,7 +10,7 @@
   $rvm_gems= {},
   $proxy_url=$rvm::params::proxy_url,
   $no_proxy=$rvm::params::no_proxy,
-  $gnupg_key_id=$rvm::params::gnupg_key_id,
+  Array[Hash[String[1], String[1]]] $signing_keys = $rvm::params::signing_keys,
```

Comment:
I did the same thing here: https://github.com/voxpupuli/puppet-rvm/pull/18, but I can easily rebase. Yay for data types.